cli/cmd/common.go

```diff
@@ -27,13 +27,32 @@ func GetApiServerUrl() string {
 }
 
 func startProxyReportErrorIfAny(kubernetesProvider *kubernetes.Provider, cancel context.CancelFunc) {
```

Comment:
lets change this function name, as its functionality changed.
something like, openConnectionToAgent(

Reply:
I don't think the funcionallity change, it still start proxy, just proxy way can be differnt but I will try to think if I have better name